tensorflow/python/lib/io/file_io_test.py

```diff
@@ -681,6 +681,17 @@ def testGetRegisteredSchemes(self):
         expected.append("gs")
     self.assertCountEqual(expected, actual)
 
+  def testReadWriteWithEncoding(self):
+    with open("text.txt", mode="w", encoding="cp932") as f:
+      f.write("今日はいい天気")
+    with file_io.FileIO("text.txt", mode="r", encoding="cp932") as f:
+      assert f.read() == "今日はいい天気"
+
+    with file_io.FileIO("text.txt", mode="w", encoding="cp932") as f:
+      f.write("今日はいい天気")
+    with open("text.txt", mode="r", encoding="cp932") as f:
+      assert f.read() == "今日はいい天気"
```

Comment:
Please switch the asserts to use `self.assertEqual` for consistency with other tests

Reply:
Thank you! I changed.